test/runtests.jl

```diff
@@ -27,8 +31,9 @@ using Test
         spl2 = StaticMH(MvNormal([0.0, 0.0], 1))
 
         # Sample from the posterior.
-        chain1 = sample(model, spl1, 100000; chain_type=StructArray, param_names=["μ", "σ"])
-        chain2 = sample(model, spl2, 100000; chain_type=StructArray, param_names=["μ", "σ"])
+        kwargs = (progress=false, chain_type=StructArray, param_names=["μ", "σ"])
+        chain1 = sample(model, spl1, 100000; kwargs...)
+        chain2 = sample(model, spl2, 100000; kwargs...)
```

Comment:
Can we maybe just define
```julia
_sample(args...; kwargs...) = sample(args...; progress = false, kwargs...)
```
at the top of the test suite and just replace all occurrences of `sample` with `_sample`? I think the `kwargs` definitions are a bit uncommon...

Reply:
Sure, I often define a named tuple for kwargs that are repeatedly used, but could be an idiosyncratic thing :)